src/EventService/EventService.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
+  <ItemGroup>
+    <ProjectReference Include="..\EventService.Broker\EventService.Broker.csproj" />   
+    <ProjectReference Include="..\EventService.Business\EventService.Business.csproj" />   
+    <ProjectReference Include="..\EventService.Data.Provider.MsSql.Ef\EventService.Data.Provider.MsSql.Ef.csproj" />
```

Comment:
don't need it

Reply:
Removed Broker, Business, Validation references